### ISSUE TITLE
[Feature Request] Keyboard shortcut to show auto-hidden (top) panel? [$5]

### ISSUE THREAD

**Issue by orschiro** [mate-desktop/mate-panel#566]
Dear all, 

I love the option to auto-hide the top panel!

Currently, the only way to show it is to go with the mouse cursor to the edge of the screen. 

I believe it would be a great enhancement if one could press the Super key to show the hidden panel.

What do you think?

Yours,

Robert

**Comment by raveit65** [comment]
super key is used by mintmenu/mate-menu

**Comment by orschiro** [comment]
Thus incompatible.

**Comment by raveit65** [comment]
Well, the idea isn't bad but we need to search for suitable keystroke combination, which doesn't result in a conflict with another application..

**Comment by orschiro** [comment]
Good point. I reopened it and changed the title to a generic keyboard shortcut.
